store/sqlstore/command_webhook_store.go

```diff
@@ -82,7 +107,17 @@ func (s SqlCommandWebhookStore) TryUse(id string, limit int) *model.AppError {
 func (s SqlCommandWebhookStore) Cleanup() {
 	mlog.Debug("Cleaning up command webhook store.")
 	exptime := model.GetMillis() - model.COMMAND_WEBHOOK_LIFETIME
-	if _, err := s.GetMaster().Exec("DELETE FROM CommandWebhooks WHERE CreateAt < :ExpTime", map[string]interface{}{"ExpTime": exptime}); err != nil {
+
+	query := s.getQueryBuilder().
+		Delete("CommandWebhooks").
+		Where(sq.Lt{"CreateAt": exptime})
+
+	queryString, args, err := query.ToSql()
+	if err != nil {
+		mlog.Error("Failed to build query.")
```

Comment:
I'd add some more context to this line and an early return in case this fails:

```suggestion
		mlog.Error("Failed to build query when trying to perform a cleanup in command webhook store.")
		return
```

Reply:
Added, thank you for the review @mgdelacroix ! :slightly_smiling_face: